shopify-cli.gemspec

```diff
@@ -0,0 +1,33 @@
+require_relative 'lib/shopify-cli/version'
+
+Gem::Specification.new do |spec|
+  spec.name = "shopify-cli"
+  spec.version = ShopifyCli::VERSION
+  spec.authors = ["Shopify"]
+  spec.email = ["developers@jadedpixel.com"]
```

Comment:
This needs to be something else. Why are you using jadedpixel and how do you even know about it!? lol 

This should probably be our developer-tool-team or whatever that group is

Reply:
I looked at the `gemspec` file of `shopify_api`!  That's what's used in there!!

Reply:
oh no